iroha/src/telemetry/dev.rs

```diff
@@ -33,19 +36,23 @@ pub fn start(
         .create(true)
         .truncate(true)
         .open(telemetry_file)
+        .await
         .wrap_err("Failed to create and open file for telemetry")?;
-    let mut writer = EncoderBuilder::new()
-        .auto_flush(true)
-        .build(file)
-        .wrap_err("Failed to create lz4 encoder")?;
```

Comment:
What is the point of removing lz4? You just said that didn't want to change semantical things in pr, as long as it is not needed for migration?

Reply:
In short it was needed because it did not support async API and was not Sync and Send to be spawned as blocking.

Reply:
May be it can be wrapped in Mutex?

Reply:
https://docs.rs/tokio/1.6.1/tokio/task/fn.spawn_blocking.html
https://docs.rs/async-std/1.9.0/async_std/task/fn.spawn_blocking.html

Tokio and async std have literally the same `spawn_blocking` api. Can you just use it and literally translate code to tokio?

Reply:
We discussed it with @i1i1 on a call and came to conclusion that it is possible to use https://crates.io/crates/async-compression there

Reply:
https://jira.hyperledger.org/browse/IR-1110